pages/heart.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: Holistic Enterprise Accessibility Review Technique (HEART)
+description: Ensuring Government Digital Services Accessibility
+permalink: /heart
+
+---
+
+![Image with the 5 phases of the HEART process](https://lh5.googleusercontent.com/7Cl86hJzMPTeYd8r312ACmsxVB0TOO0bMpoQYil2ooYPEnaFwTZ_yTc-TJ2W_ZxYOWwhd8MsXUCYH1GP5umfUmWaj_7gZut3t9mFTGaeI3L_iKMFk7tbVdoLfY6_mxV4lCtDKTZ5)
```

Comment:
@mgifford should refer to a local copy of this image so that we don't have to worry about it disappearing?

Reply:
Absolutely. I've modified this to refer to the local heart.png file.